File: src/classes/api/req_res/get-mi-board-response.ts
// ˅
'use strict';

import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class GetMiBoardResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    boards: Array<string>;

    constructor() {
        // ˅
        super()
        this.boards = new Array<string>()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/server/gkill/api/req_res/get_mi_board_response.go
package req_res

import "github.com/mt3hr/gkill/src/server/gkill/api/message"

type GetMiBoardResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	Boards []string `json:"boards"`
}
